From 0f82fac0e9403be739ba9fd8100b1e48bc70b20a Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Thu, 4 Jul 2024 23:01:07 +0800
Subject: [PATCH 2/2] fix(astyle_main.cpp): fix parseOption() usage against a
 recent change

This method used to take two parameters, now it only takes one.

Fixes: 5146121caab719a9d3884d46305582a0c429f324
---
 AStyle/src/astyle_main.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/AStyle/src/astyle_main.cpp b/AStyle/src/astyle_main.cpp
index 40f16873..354a6a42 100644
--- a/AStyle/src/astyle_main.cpp
+++ b/AStyle/src/astyle_main.cpp
@@ -4213,7 +4213,7 @@ extern "C" EXPORT char* STDCALL AStyleMain(const char* pSourceIn,		// the source
 
 	options.importOptions(opt, optionsVector);
 
-	bool ok = options.parseOptions(optionsVector, "Invalid Artistic Style options:");
+	bool ok = options.parseOptions(optionsVector);
 	if (!ok)
 		fpErrorHandler(130, options.getOptionErrors().c_str());
 
-- 
2.45.2

